src/components/change-wallet/AddressRow.js

```diff
@@ -169,21 +161,17 @@ export default function AddressRow({
               />
             ) : (
               <ContactAvatar
-                color={accountColor}
+                address={address}
+                color={color}
+                emoji={emoji}
                 marginRight={10}
                 size="medium"
-                value={
-                  returnStringFirstEmoji(label) ||
-                  profileUtils.addressHashedEmoji(address) ||
-                  label ||
-                  ens
-                }
               />
             )}
             <ColumnWithMargins margin={android ? -6 : 3}>
-              {cleanedUpLabel || ens ? (
+              {label || ens ? (
                 <StyledTruncatedText color={colors.dark}>
-                  {cleanedUpLabel || ens}
+                  {label || ens}
```

Comment:
this is more like an open question, but what do you thing of making the ens the default value? and if there's no ens let's show the label

Reply:
i think it could be a good idea to only allow users to give wallets a nickname if the wallet doesn't have a primary ens

@christianbaroni wdyt? i remember we talked about this before